### PR TITLE
Adds the new build from func-e

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Notably, this:
* stops using bingo and goreleaser
* adds an issue template
* creates nice template release notes
* re-uses the go installation from GitHub Actions runner
* updates to go 1.17
* colorizes make output

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
The incentive to do this maintenance is that we've not yet released a version of car. If we do, our runners can work easier as they don't need to install go in order to run car!

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# `name` value will appear "as is" in the badge.
+# See https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository
+# yamllint --format github .github/workflows/release.yaml
+---
+name: "release"
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+**'  # Ex. v0.2.0 v0.2.1-rc2
```
Comment (jcchavezs) [inline .github/workflows/release.yaml:9]: Shall you ass `^`?

Comment (codefromthecrypt) [reply .github/workflows/release.yaml:9]: yeah, let's ass that!

File: Makefile [hunk 2]
```diff
@@ -1,91 +1,167 @@
...
+	@printf "$(ansi_format_dark)" sha256sum "generating $@"
+	@$(sha256sum) $^ > $@
+	@printf "$(ansi_format_bright)" sha256sum "ok"
+
+# dist generates the assets that attach to a release
+# Ex. https://github.com/tetratelabs/car/releases/tag/v$(VERSION)
+dist: $(archives) $(packages) $(checksums) ## Generate release assets
+
+clean: ## Ensure a clean build
+	@printf "$(ansi_format_dark)" clean "deleting temporary files"
```
Comment (jcchavezs) [inline Makefile:117]: nit: `@printf "$(ansi_format_xxxx)"` seems repetitive. Can't be in a func like `xxxx`??

File: .github/workflows/release.yaml (hunk 1 above)
Comment (codefromthecrypt) [reply .github/workflows/release.yaml:9]: e64a8239a0ad03ef59e84328a29800347bde15e5

**Pull request merged**